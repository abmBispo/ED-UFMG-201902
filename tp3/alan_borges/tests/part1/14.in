1024
e sim tem alguma cousa os olhos a disposicao do rosto e o pai um pouco mais moderno concluiu por chalaca e diga me agora mana gloria nao foi melhor que ele nao teimasse em ser padres veja se este peralta daria um padre capaz como vai o meu substituto vai indo ordena se para o ano respondeu tio cosme has de ir ver a ordenacao eu tambem se o meu senhor coracao consentir e bom que te sintas na alma do outro como se recebesses em ti mesmo a sagracao justamente exclamou minha mae mas veja bem mano cosme veja se nao e a figura do meu defunto olha bentinho olha bem para mim sempre achei que te parecias com ele agora e muito mais o bigode e que desfaz um pouco sim mana gloria o bigode realmente mas e muito parecido e minha mae beijava me com uma ternura que nao sei escrever tio cosme para alegra la chamava me doutor jose dias tambem e todos em casa a prima os escravos as visitas padua a filha e ela mesma repetiam me o titulo capitulo c tu seras feliz bentinho no quarto desfazendo a mala e tirando a carta de bacharel de dentro da lata ia pensando na felicidade e na gloria via o casamento e a carreira ilustre enquanto jose dias me ajudava calado e zeloso uma fada invisivel desceu ali e me disse em voz igualmente macia e calida tu seras feliz bentinho tu vais ser feliz e por que nao seria feliz perguntou jose dias endireitando o tronco e fitando me voce ouviu perguntei eu erguendo me tambem espanta ouviu o que ouviu uma voz que dizia que eu serei feliz e boa voce mesmo e que esta dizendo ainda agora sou capaz de jurar que a voz era da fada naturalmente as fadas expulsas dos contos e dos versos meteram se no coracao da gente e falam de dentro para fora esta por exemplo muita vez a ouvi clara e distinta ha de ser prima das feiticeiras da escocia tu seras rei macbeth tu seras feliz bentinho ao cabo e a mesma predicao pela mesma toada universal e eterna quando voltei do meu espanto ouvi o resto do discurso de jose dias ha de ser feliz como merece assim como mereceu esse diploma que ali esta que nao e favor de ninguem a distincao que tirou em todas as materias e prova disso ja lhe contei que ouvi da boca dos lentes em particular os maiores elogios demais a felicidade nao e so a gloria e tambem outra cousa ah voce nao confiou tudo ao velho jose dias o pobre jose dias esta ai para um canto e caju chupado nao vale nada agora sao os novos os escobares nao lhe nego que e moco muito distinto e trabalhador e marido de truz mas enfim velho tambem sabe amar mas que e que ha de ser quem e que nao sabe tudo aquela intimidade de vizinhos tinha de acabar nisto que e verdadeiramente uma bencao do ceu porque ela e um anjo e um anjissimo perdoe a cincada bentinho foi um modo de acentuar a perfeicao daquela moca cuidei o contrario outrora confundi os modos de crianca com expressoes de carater e nao vi que essa menina travessa e ja de olhos pensativos era a flor caprichosa de um fruto sadio e doce por que e que nao me contou tambem o que outros sabem e ca em casa esta mais que adivinhado e aprovado mamae aprova deveras pois entao temos falado sobre isso e ela fez me o favor de pedir a minha opiniao pergunte lhe o que e que eu lhe disse em termos claros e positivos pergunte lhe disse lhe que nao podia desejar melhor nora para si boa discreta prendada amiga da gente e uma dona de casa que nao lhe digo nada depois da morte da mae tomou conta de tudo padua agora que se aposentou nao faz mais que receber o ordenado e entrega lo a filha a filha e que distribui o dinheiro paga as contas faz o rol das despesas cuida de tudo mantimento roupa luz voce ja a viu o ano passado e quanto a formosura voce sabe melhor que ninguem mas deveras mamae consultou o senhor sobre o nosso casamento positivamente nao fez me o favor de perguntar se capitu nao daria uma boa esposa eu e que na resposta falei em nora d gloria nao negou e ate deu um ar de riso mamae sempre que me escrevia falava de capitu voce sabe que elas se dao muito e por isso e que sua prima anda cada vez mais amuada talvez agora case mais depressa prima justina nao sabe sao contos naturalmente mas enfim o doutor joao da costa enviuvou ha poucos meses e dizem nao sei o protonotario e que me contou dizem que os dous andam meio inclinados a acabar com a viuvez entre si casando se ha de ver que nao ha nada mas nao e fora de proposito contanto ela sempre achasse que o doutor era um feixe de ossos so se ela e um cemiterio comentou rindo e logo serio digo isto por gracejo nao ouvi o resto ouvia so a voz da minha fada interior que me repetia mas ja entao sem palavras tu seras feliz bentinho e a voz de capitu me disse a mesma cousa com termos diversos e assim tambem a de escobar os quais ambos me confirmaram a noticia de jose dias pela sua propria impressao enfim minha mae algumas semanas depois quando lhe fui pedir licenca para casar alem do consentimento deu me igual profecia salva a redacao propria de mae tu seras feliz meu filho capitulo cii no ceu pois sejamos felizes de uma vez antes que o leitor pegue em si morto de esperar e va espairecer a outra parte casemo nos foi em uma tarde de marco por sinal que chovia quando chegamos ao alto da tijuca onde era o nosso ninho de noivos o ceu recolheu a chuva e acendeu as estrelas
q acendeu
q acentuar
q achasse
q achei
q adivinhado
q ah
q ai
q ainda
q ajudava
q alegra
q alem
q alguma
q algumas
q alma
q alto
q amar
q ambos
q amiga
q amuada
q anda
q andam
q anjissimo
q anjo
q antes
q aposentou
q aprova
q aprovado
q aquela
q ar
q ate
q bacharel
q beijava
q bencao
q boca
q bom
q c
q ca
q cabo
q cada
q caju
q calado
q calida
q canto
q caprichosa
q carater
q carreira
q carta
q casando
q casar
q case
q casemo
q cemiterio
q chalaca
q chamava
q chegamos
q chovia
q chupado
q chuva
q cii
q cincada
q clara
q claros
q comentou
q concluiu
q confiou
q confirmaram
q confundi
q consentimento
q consentir
q consultou
q conta
q contanto
q contas
q contei
q contrario
q costa
q crianca
q cuida
q cuidei
q d
q dao
q daquela
q defunto
q demais
q depressa
q desceu
q desejar
q desfaz
q desfazendo
q despesas
q diga
q dinheiro
q diploma
q discreta
q discurso
q disposicao
q disso
q distincao
q distinta
q distinto
q distribui
q diversos
q dizendo
q dizia
q doce
q dona
q dous
q elas
q elogios
q endireitando
q enquanto
q entre
q entrega
q enviuvou
q erguendo
q escobar
q escobares
q escocia
q escravos
q escrever
q escrevia
q espairecer
q espanta
q espanto
q esperar
q esposa
q essa
q esse
q este
q estrelas
q eterna
q exclamou
q exemplo
q expressoes
q expulsas
q fadas
q falado
q falam
q falava
q falei
q feiticeiras
q feixe
q felizes
q figura
q filho
q fitando
q flor
q formosura
q fruto
q fui
q gracejo
q has
q ia
q igual
q igualmente
q ilustre
q impressao
q inclinados
q indo
q interior
q intimidade
q invisivel
q ir
q isto
q joao
q jurar
q justamente
q justina
q la
q lata
q leitor
q lentes
q licenca
q lo
q logo
q luz
q macbeth
q macia
q maiores
q mala
q mano
q mantimento
q marco
q marido
q materias
q meio
q menina
q merece
q mereceu
q meses
q meteram
q mim
q moca
q moco
q moderno
q modo
q modos
q morte
q morto
q muita
q nego
q negou
q ninho
q nisto
q noivos
q nos
q noticia
q novos
q onde
q opiniao
q ordena
q ordenacao
q ordenado
q ossos
q outro
q outrora
q outros
q ouvia
q padre
q padres
q paga
q pai
q palavras
q parecias
q parecido
q parte
q particular
q passado
q pegue
q pensando
q pensativos
q peralta
q perdoe
q perfeicao
q perguntar
q perguntei
q perguntou
q pobre
q podia
q porque
q positivamente
q positivos
q poucos
q predicao
q prendada
q profecia
q proposito
q protonotario
q prova
q quais
q quanto
q quarto
q quem
q realmente
q receber
q recebesses
q recolheu
q redacao
q rei
q repetia
q repetiam
q respondeu
q resposta
q rindo
q riso
q rol
q rosto
q roupa
q sabem
q sadio
q sagracao
q salva
q sejamos
q sem
q semanas
q serei
q seria
q serio
q sinal
q sintas
q sou
q substituto
q talvez
q tarde
q teimasse
q tem
q temos
q ternura
q ti
q tijuca
q tinha
q tirando
q tirou
q titulo
q toada
q todas
q todos
q tomou
q trabalhador
q travessa
q tronco
q truz
q universal
q va
q vais
q vale
q verdadeiramente
q versos
q vi
q via
q visitas
q viu
q viuvez
q vizinhos
q voltei
q zeloso
q acabar
q ali
q ano
q assim
q bem
q bigode
q capaz
q capitulo
q casamento
q contos
q contou
q coracao
q daria
q das
q dentro
q depois
q deu
q deveras
q digo
q dizem
q ele
q entao
q faz
q felicidade
q fez
q fora
q gente
q isso
q mana
q mesmo
q naturalmente
q ninguem
q nora
q nosso
q olha
q olhos
q outra
q padua
q pedir
q pela
q pergunte
q pois
q pouco
q propria
q resto
q sao
q sei
q senhor
q sim
q sobre
q sua
q te
q termos
q tio
q vai
q velho
q ver
q ao
q boa
q capitu
q casa
q ceu
q cosme
q cousa
q dos
q doutor
q enfim
q fada
q favor
q filha
q foi
q mamae
q melhor
q nada
q no
q ouviu
q quando
q sempre
q si
q so
q veja
q vez
q como
q disse
q era
q ja
q mesma
q muito
q na
q ouvi
q prima
q tudo
q as
q com
q ela
q esta
q eu
q gloria
q mae
q meu
q minha
q sabe
q ser
q voz
q agora
q bentinho
q ha
q mais
q seras
q voce
q dias
q do
q jose
q para
q por
q tambem
q tu
q lhe
q mas
q os
q uma
q feliz
q da
q em
q se
q um
q me
q nao
q o
q a
q de
q que
q e
