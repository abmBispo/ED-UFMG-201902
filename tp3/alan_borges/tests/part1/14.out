1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
3
4
4
4
4
4
4
4
4
4
4
5
5
5
5
5
5
5
5
5
5
5
5
6
6
6
6
6
6
7
7
7
7
7
7
7
8
8
8
8
9
11
11
11
11
17
24
30
33
36
44
65
